### PR TITLE
CLB traffic to instances in the same subnet

### PR DESCRIPTION
**Pull request by kbusekist**
Traffic between CLBs and Instances in the same subnet do not behave the same was as Instance to Instance traffic in the same subnet.

### CODE REVIEW

**Comment by ThanosBaskous** [comment]
Great addition! Thanks @kbusekist!

@jlevy would you mind checking this out for final review/merge?

File: README.md [hunk 1]
```diff
@@ -858,6 +858,7 @@ CLB
...
 -	**Apex DNS names:** Once upon a time, you couldn’t assign an CLB to an apex DNS record (i.e. example.com instead of foo.example.com) because it needed to be an A record instead of a CNAME. This is now possible with a Route 53 alias record directly pointing to the load balancer.
 -	🔸CLBs use [HTTP keep-alives](https://en.wikipedia.org/wiki/HTTP_persistent_connection) on the internal side. This can cause an unexpected side effect: Requests from different clients, each in their own TCP connection on the external side, can end up on the same TCP connection on the internal side. Never assume that multiple requests on the same TCP connection are from the same client!
+-	Traffic between CLBs and back-end instances in the same subnet **will** have NACL rules evaluated (EC2 to EC2 traffic in the same subnet would not have NACL rules evaluated). If the default '0.0.0.0/0 ALLOW' rule is removed from the NACL applied to the subnet, a rule that allows traffic on both the health check port and any listener port must be added.
```
Comment (jlevy) [inline README.md]: I believe AWS standard term is "Network ACL". Can we use that and link to the page where this term first appears in the sentence?

Also, add 🔸.

**Comment by jlevy** [comment]
Thanks!

**Comment by richadams** [comment]
LGTM 👍

**Comment by jlevy** [comment]
👍 Thanks all!

**Pull request merged**